http-apps/cookie.js

```diff
@@ -14,53 +14,49 @@ exports.CookieJar = function (app) {
 
         var now = new Date();
 
-        var requestCookies = concat(hosts.map(function (host) {
```

Comment:
In the [no-whitespace diff](https://github.com/kriskowal/q-io/pull/128/files?w=0) this change is small. Reading through I can't tell if the logic is correct, I would need to step through this and read the cookie spec.

Removing the `.map` does leave the `hosts` variable unused.